src/components/popover.ts

```diff
@@ -7,7 +7,7 @@ const { xml } = tags;
 
 const TEMPLATE = xml/* xml */ `
   <Portal target="'.o-spreadsheet'">
-    <div t-att-style="style">
+    <div t-att-style="style" t-attf-class="o-popover">
```

Comment:
```suggestion
    <div class="o-popover" t-att-style="style">
```
useless and harder to read to use a `t-attf` for something static

Reply:
Changes made